lib/embed.dart

```diff
@@ -375,7 +375,7 @@ class Embed {
       });
 
     if (options.mode == EmbedMode.flutter || options.mode == EmbedMode.html) {
-      consoleExpandController = ConsoleExpandController(
+      var controller = ConsoleExpandController(
```

Comment:
I'm missing something here. What's the reason for creating the local `controller`?

Reply:
The `consoleExpandController` needs to be a `Console` (the superclass) because can be either a `ConsoleExpandController` or a `Console`. Since the `open()` method is only defined on the ConsoleExpandController, we can only call it while we _know_ that it's a `ConsoleExpandController`.

Reply:
Got it.